OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -0,0 +1,308 @@
+# Oracle WebCenter Portal 12.2.1.4.0 on Docker
+
+Before running an Oracle WebCenter Portal image below security configuration are recommended to be followed.
+## Docker Security Configuration
+
+### A. Configuring auditing rules for docker 
```

Comment:
What is the purpose of this?

Reply:
This are Security configuration which are recommended to be followed . As part of Security Bug fix we have added it